web/src/main/java/org/cbioportal/web/parameter/GeneFilter.java

```diff
@@ -0,0 +1,125 @@
+package org.cbioportal.web.parameter;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
```

Comment:
delete extra blank line.

Reply:
Can we collapse these into java.util.*, java.util.regex.*...